libc/string_lenchr.c

```diff
@@ -0,0 +1,210 @@
+/*
+ * Phoenix-RTOS
+ *
+ * libc-tests
+ *
+ * libphoenix strlen and strnlen tests
+ *
+ * Copyright 2021, 2023 Phoenix Systems
+ * Author: Damian Modzelewski
+ *
+ * This file is part of Phoenix-RTOS.
+ *
+ * %LICENSE%
+ */
+
+
+#include <stdio.h>
+#include <errno.h>
+#include <string.h>
+#include <limits.h>
+#include <stdlib.h>
+#include <sys/stat.h>
+#include <unistd.h>
+#include <limits.h>
+
+#include <unity_fixture.h>
+
+
+TEST_GROUP(string_strlen);
+
+
+TEST_SETUP(string_strlen)
+{
+}
+
+
+TEST_TEAR_DOWN(string_strlen)
+{
+}
+
+
+TEST(string_strlen, strlen_right_input)
```

Comment:
I like this name for some simple checks, but maybe the others have a better idea.

```suggestion
TEST(string_strlen, strlen_simple)
```